graphql/README.md

```diff
@@ -0,0 +1,128 @@
+## GraphQL Introduction
+
+### Getting Started
+
+To get the GraphQL server started, type `npm run start-graphql` in the terminal and go to localhost:3001/graphql.
+
+To get the endpoints that are currently available, click on the "Docs" tab on the right hand side of the screen to open the documentation explorer. The documentation explorer allows you to click through the different available queries, display the type associated with that query, and the schema for each of the types.
+
+<img src="documentation_images/doc explorer.gif" width="300px"/>
+
+You can open a new tab inside the interface to make another query without losing your current one.
+
+<img src="documentation_images/newtab.gif" width="300px" />
+
+---
+
+### Querying
+
+Unlike RESTful APIs, GraphQL allows the client to determine what information the server returns. For example, our NumberType _can_ pass back facts about the number as types: trivia, date, math, or year. But if the user only wants the math fact, they can specify only the math fact should be passed back.
+
+Example queries (user puts in number in place of int):
+
+Get all facts about a specific number
+
+```
+query {
+  number(number: int) {
```

Comment:
I am getting an error when running this query. Did the request format change?

Reply:
Updated the documentation to show the correct query endpoint (number -> getNumberFacts)